test/unit/components/provider.tsx

```diff
@@ -93,6 +94,8 @@ describe('<IntlProvider>', () => {
   });
 
   it('provides `context.intl` with values from intl config props', () => {
+    areIntlLocalesSupported.mockClear();
+    areIntlLocalesSupported.mockReturnValue(true);
```

Comment:
We just return `true`, effectively whitelisting all the languages. I think for this kind of test this is totally fine.